stdlib/array.mli

```diff
@@ -220,8 +226,7 @@ val sort : ('a -> 'a -> int) -> 'a array -> unit
    compare as equal, a positive integer if the first is greater,
    and a negative integer if the first is smaller (see below for a
    complete specification).  For example, {!Stdlib.compare} is
-   a suitable comparison function, provided there are no floating-point
-   NaN values in the data.  After calling [Array.sort], the
+   a suitable comparison function.  After calling [Array.sort], the
```

Comment:
(Why not keep this sentence? I guess developers
are likely to use `float array` value for some time,
*e.g.* to interact with legacy libraries.)

Reply:
In fact `Stdlib.compare` is suitable even if there are NaN values.